src/redux/actions.ts

```diff
@@ -4,6 +4,4 @@ function changeSection(sectionName: SectionName): AppAction {
   return { type: CHANGE_SECTION, sectionName };
 }
 
-export default {
-  changeSection,
-};
+export default changeSection;
```

Comment:
im assuming we're going to have more than one function in here in the future in which case we should just declare each as a closure:
```
export const changeSection = (sectionName: SectionName) => {

}
```

so that to import it, we can do `import { changeSection, <any other functions> } from '/actions'